test/rails5.0/app/cells/formtastic_cell.rb

```diff
@@ -1,4 +1,6 @@
 class FormtasticCell < Cell::ViewModel
+  view_paths << "#{Rails.root}/app/cells"
```

Comment:
I'm adding because view paths has the default value

* app/cells

and with the command added, it adds the following path:

* /Users/fernandes/src/upstream/cells-rails/test/rails4.2/app/cells

maybe it's something with the `dummy` app